vectorstores/alloydb/vectorstore_test.go

```diff
@@ -57,38 +59,38 @@ func setEngine(t *testing.T) (alloydbutil.PostgresEngine, error) {
 		alloydbutil.WithAlloyDBInstance(projectID, region, cluster, instance),
 	)
 	if err != nil {
-		t.Fatal("Could not set Engine: ", err)
+		return alloydbutil.PostgresEngine{}, err
```

Comment:
@kurtisvg is this correct for a testing helper method?

Reply:
Generally it should either use testing.T or have a return type. 

In this case since it's returning an error I would suggest leaving this line alone but removing testing.T as input and removing t.Helper().